Makefile

```diff
@@ -33,3 +33,7 @@ run: ## Run project locally
 .PHONY: docs
 docs: ## Generate and show documentation
 	cargo doc --open 
+
+.PHONY: screencast
+screencast: ## Create a screencast for the docs
+	svg-term --command="bash assets/screencast.sh" --out assets/screencast.svg --padding=10 --window --height 80 --width 120
```

Comment:
I suggest to make the width smaller. Currently most of the horizontal spaces is unused and fonts are too small.

Reply:
Can you check again? I think it looks better now. I will update the `Makefile` accordingly if you like it.

Reply:
It's readable now.

Reply:
Updated Markdown file